fdbserver/Knobs.cpp

```diff
@@ -364,7 +364,7 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( MAX_PROXY_COMPUTE,                                      2.0 );
 	init( PROXY_COMPUTE_BUCKETS,                                20000 );
 	init( PROXY_COMPUTE_GROWTH_RATE,                             0.01 );
-	init( TXN_STATE_SEND_AMOUNT,                                    2 );
+	init( TXN_STATE_SEND_AMOUNT,                                    4 );
```

Comment:
It may be better to buggify these knobs to improve confidence when Ops changes the value.